src/Keboola/GoogleAnalyticsExtractor/Extractor/Output.php

```diff
@@ -64,16 +62,52 @@ public function createReport($query)
             return str_replace('ga:', '', $item['expression']);
         }, $query['query']['metrics']);
 
-        $headerRow = array_merge(
+        return array_merge(
             ['id', 'idProfile'],
             $dimensions,
             $metrics
         );
-        $csv->writeRow($headerRow);
+    }
 
+    public function createReport($query)
+    {
+        $csv = $this->createCsvFile(sprintf('%s_%s', $query['outputTable'], uniqid()));
+        $csv->writeRow($this->createHeaderRowFromQuery($query));
         return $csv;
     }
 
+    public function createJsonSampleReport($query, $report)
```

Comment:
A toto by som radsej nazval `createReportJson` alebo tiez `createSampleReportJson` aby to bolo konzistentne v ramci tejto triedy.